### PR TITLE
fine tune LS to be run for data WFs

### PR DESCRIPTION
**Pull request by sunilUIET**
#### PR description:

This PR is fine tune/fixing a few settings for data WFs
1) Lumi-sections optimized for CPU-friendly but sufficient statistics for release validations
2) Addition of @standardDQM in Harvesting step along with era Run3
3) Lumis-per-job for different steps

#### PR validation:

However, changes are straight forward but a local test is performed with WF ID 140.042

### CODE REVIEW

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -3344,7 +3349,7 @@ def gen2021HiMix(fragment,howMuch):
 steps['HARVESTPROMPTRUN3']=merge([{'--data':'', '-s':'HARVESTING:@allForPrompt','--era':'Run3'},steps['HARVESTDRUN3']])
 
 
-steps['HARVESTRUN3']=merge([{'--data':'', '-s':'HARVESTING:@miniAODDQM+@nanoAODDQM'},steps['HARVESTDRUN3']])
+steps['HARVESTRUN3']=merge([{'--data':'', '-s':'HARVESTING:@standardDQM+@miniAODDQM+@nanoAODDQM','--era':'Run3'},steps['HARVESTDRUN3']])
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_steps.py:3352]: Are you sure on `HARVESTING:@standardDQM` ?
Does it not look on `TriggerResults::HLT` which is not what you expect, i.e. relvals workflow run with reHLT, i.e. https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_standard.py#L452

Comment (sunilUIET) [reply Configuration/PyReleaseValidation/python/relval_steps.py:3352]: Thanks Phat! Then which DQM sequence will be an appropriate choice?

Comment (srimanob) [reply Configuration/PyReleaseValidation/python/relval_steps.py:3352]: Hi @sunilUIET 

Sorry, I don't know. The above is only my guess from skimming quickly on configuration. I may be wrong.
You may clarify with @cms-sw/dqm-l2 @cms-sw/hlt-l2

**Comment by sunilUIET** [comment]
Hi @srimanob , In Run 2 we used a setup with standardDQMFakeHLT, so we can use the same sequence for Run 3.

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
+Upgrade

**Comment by sunilUIET** [comment]
+pdmv

**Comment by rappoccio** [comment]
+1

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 2]
```diff
@@ -2781,7 +2786,7 @@ def gen2021HiMix(fragment,howMuch):
 steps['RECODR3_reHLT_HLTPhysics_Offline']=merge([{'-s':'RAW2DIGI,L1Reco,RECO,PAT,ALCA:TkAlMinBias+HcalCalIterativePhiSym+HcalCalIsoTrkProducerFilter+HcalCalHO+HcalCalHBHEMuonProducerFilter,DQM:@commonReduced+@miniAODDQM','--procModifiers':'siPixelQualityRawToDigi'},steps['RECODR3_reHLT']])
 steps['RECODR3_reHLT_AlCaTkCosmics_Offline']=merge([{'-s':'RAW2DIGI,L1Reco,RECO,SKIM:EXONoBPTXSkim,PAT,ALCA:TkAlCosmicsInCollisions,DQM:@standardDQMFakeHLT+@miniAODDQM'},steps['RECODR3_reHLT']])
 
-steps['RECONANORUN3_reHLT']=merge([{'-s':'RAW2DIGI,L1Reco,RECO,PAT,NANO:PhysicsTools/NanoAOD/V10/nano_cff,DQM:@standardDQM+@miniAODDQM+@nanoAODDQM'},steps['RECODR3_reHLT']])
+steps['RECONANORUN3_reHLT']=merge([{'-s':'RAW2DIGI,L1Reco,RECO,PAT,NANO:PhysicsTools/NanoAOD/V10/nano_cff,DQM:@standardDQMFakeHLT+@miniAODDQM+@nanoAODDQM'},steps['RECODR3_reHLT']])
```
Comment (mmusich) [inline Configuration/PyReleaseValidation/python/relval_steps.py:2789]: This makes not sense. If it is a re-HLT workflow we *do* want to look at the HLT monitoring!

Comment (srimanob) [reply Configuration/PyReleaseValidation/python/relval_steps.py:2789]: Hi @mmusich 
Thanks for the comment. Please open the PR. Thx.

Comment (mmusich) [reply Configuration/PyReleaseValidation/python/relval_steps.py:2789]: Hi @srimanob, see https://github.com/cms-sw/cmssw/pull/40804

**Pull request merged**